src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,236 @@
+} from "../../../shared/model/MatchMiner";
+import styles from './style/trialMatch.module.scss';
+import { computed } from "mobx";
+import LazyMobXTable from "../../../shared/components/lazyMobXTable/LazyMobXTable";
+import SampleManager from "../sampleManager";
+import DefaultTooltip, { placeArrowBottomLeft } from "../../../public-lib/components/defaultTooltip/DefaultTooltip";
+
+export type ITrialMatchProps = {
+    sampleManager: SampleManager | null;
+    detailedTrialMatches: IDetailedTrialMatch[];
+    containerWidth: number;
+}
+
+enum ColumnKey {
+    ID = 'ID',
+    TITLE = 'Title',
+    MATCHING_CRITERIA = 'Matching Criteria'
+}
+
+enum ColumnWidth {
+    ID = 140
+}
+
+class TrialMatchTableComponent extends LazyMobXTable<IDetailedTrialMatch> {
+
+}
+
+@observer
+export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
+
+    @computed
+    get columnsWidth() {
+        return {
+            [ColumnKey.ID]: ColumnWidth.ID,
+            [ColumnKey.TITLE]: 0.35 * (this.props.containerWidth - ColumnWidth.ID),
+            [ColumnKey.MATCHING_CRITERIA]: 0.65 * (this.props.containerWidth - ColumnWidth.ID)
+        };
+    }
+
+    private _columns = [{
+        name: ColumnKey.ID,
+        render: (trial: IDetailedTrialMatch) => (
+            <div>
+                <If condition={trial.protocolNo.length > 0}>
+                    <div><a target="_blank" href={"https://www.mskcc.org/cancer-care/clinical-trials/" + trial.protocolNo}>{trial.protocolNo}</a></div>
+                </If>
+                <If condition={trial.nctId.length > 0}>
+                    <div><a target="_blank" href={"https://clinicaltrials.gov/ct2/show/" + trial.nctId}>{trial.nctId}</a></div>
+                </If>
+                <div>{trial.status}</div>
+            </div>
+        ),
+        sortBy: (trial: IDetailedTrialMatch) => trial.protocolNo,
+        width: this.columnsWidth[ColumnKey.ID]
+    }, {
+        name: ColumnKey.TITLE,
+        render: (trial: IDetailedTrialMatch) => (<span>{trial.shortTitle}</span>),
+        sortBy: (trial: IDetailedTrialMatch) => trial.shortTitle,
+        width: this.columnsWidth[ColumnKey.TITLE]
+    }, {
+        name: ColumnKey.MATCHING_CRITERIA,
+        render: (trial: IDetailedTrialMatch) => (
+            <div>
+                {trial.matches.map((armMatch: IArmMatch, index: number) => (
+                    <div>
+                        <div>
+                            <div>
+                                {armMatch.matches.map((clinicalGroupMatch: IClinicalGroupMatch, cgIndex:number) => (
+                                    <span className={styles.criteriaContainer}>
+                                        {this.getGenomicMatch(clinicalGroupMatch)}
+                                        {this.getClinicalMatch(clinicalGroupMatch)}
+                                        <If condition={cgIndex < armMatch.matches.length - 1}><hr className={styles.criteriaHr}/></If>
```

Comment:
you could hide the last <hr> with css by using hr:last-child css selector

Reply:
I tried but it doesn't work at here. The `:last-child` selector matches every element that is the last child of its parent. Each `<hr>` is wrapped by `<div>`, so it's always the last child of its parent `<div>`.